app/models/edition/national_applicability.rb

```diff
@@ -10,13 +10,23 @@ def process_associations_before_save(edition)
   end
 
   included do
+    validate :applicability_options
     has_many :nation_inapplicabilities, foreign_key: :edition_id, dependent: :destroy, autosave: true
     validates_associated :nation_inapplicabilities
     validates :nation_inapplicabilities, length: { maximum: Nation.all.count - 1, message: "can not exclude all nations" }
 
     add_trait Trait
   end
 
+  def applicability_options
+    if !all_nation_applicability && nation_inapplicabilities.none? {|ni| ni.excluded?}
```

Comment:
```suggestion
    if !all_nation_applicability && nation_inapplicabilities.none?(&:excluded?)
```

I think this is the preferred style: https://www.rubydoc.info/gems/rubocop/0.26.1/RuboCop/Cop/Style/SymbolProc

Reply:
👍